os/net/lwip/configs/icmp6/Kconfig

```diff
@@ -3,9 +3,10 @@
 # see kconfig-language at https://www.kernel.org/doc/Documentation/kbuild/kconfig-language.txt
 #
 
-menu "ICMP6 Support"
+menu "ICMPv6 (RFC 4443)"
 
 config NET_IPv6_ICMP
```

Comment:
same as previous

Reply:
for compatibility. there are many files to do fix.  and it's not fully tested. I'll remove it later